sdk/tests/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ add_executable(${TEST_PROJECT_NAME}
         TransferTransactionTest.cc)
 
 target_link_libraries(${TEST_PROJECT_NAME} PRIVATE gtest_main gtest ${PROJECT_NAME})
+target_link_libraries(${TEST_PROJECT_NAME} PRIVATE gmock_main gmock ${PROJECT_NAME})
```

Comment:
Are you actually using `gmock` anywhere in these tests? `gmock` should be used when we want to specifically control the response back from the network, and it looks like we aren't doing that in these tests.

Reply:
Currently I'm not using it anywhere. I will remove it as it's redundant at the moment.

Reply:
Removed in 614ab1220cb58ad90be6bfc4870a2f43a61ddb56